internal/context/snssai.go

```diff
@@ -13,7 +14,7 @@ type SNssai struct {
 
 // Equal return true if two S-NSSAI is equal
 func (s *SNssai) Equal(target *SNssai) bool {
-	return s.Sst == target.Sst && s.Sd == target.Sd
+	return s.Sst == target.Sst && strings.EqualFold(s.Sd, target.Sd)
```

Comment:
modify to `return openapi.SnssaiEqualFold(s, target)`

Reply:
`s` and `target` are also `SNssai` defined in internal/context/snssai.go instead of `model.snssai`.
Should I make the conversion?

Reply:
add 
```
func (s *SNssai) EqualModelsSnssai(target *models.Snssai) bool {
	return s.Sst == target.Sst && strings.EqualFold(s.Sd, target.Sd)
}
```